Dockerfile

```diff
@@ -182,6 +182,10 @@ RUN if [ "${CVAT_DEBUG_ENABLED}" = 'yes' ]; then \
         python3 -m pip install --no-cache-dir debugpy; \
     fi
 
+# Removing pip due to security reasons. See: https://scout.docker.com/vulnerabilities/id/CVE-2018-20225
+RUN /bin/bash -c "source /opt/venv/bin/activate && pip uninstall pip -y"
+RUN apt-get remove -y python3-pip
```

Comment:
I don't think this package is installed in this image in the first place.

Reply:
removed the unnecessary deletion